packages/client-react-streaming/src/DataTransportAbstraction/WrappedApolloClient.tsx

```diff
@@ -120,6 +100,19 @@ class ApolloClientBase extends OrigApolloClient<NormalizedCacheObject> {
       info,
       "InMemoryCache"
     );
+
+    this.setLink(this.link);
+  }
+
+  setLink(newLink: ApolloLink) {
+    super.setLink.call(
+      this,
+      ApolloLink.from([
+        new ReadFromReadableStreamLink(),
+        new TeeToReadableStreamLink(),
+        newLink,
+      ])
+    );
   }
```

Comment:
All implementations (SSR, RSC, Browser) now use the same links - RSC *could* omit the `ReadFromReadableStreamLink`, but keeping all those separate details is very prone to subtle bugs; this is more reasonable.